docs/src/app/components/pages/components/GridList/ExampleComplex.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import GridList from 'material-ui/lib/grid-list/grid-list';
+import GridTile from 'material-ui/lib/grid-list/grid-tile';
+import StarBorder from 'material-ui/lib/svg-icons/toggle/star-border';
+import IconButton from 'icon-button';
```

Comment:
That won't work when people will copy past the example.
Use the `material-ui/lib/...`